nyxx/lib/src/utils/EntityUtility.dart

```diff
@@ -102,4 +102,28 @@ class EntityUtility {
   static Member createGuildMember(
           INyxx client, Snowflake guildId, Map<String, dynamic> rawJson) =>
       Member._new(client, rawJson, guildId);
+
+  /// Creates a Guild Member object, can be used for other classes where you have correct rawJson data from the API.
+  /// ```dart
+  /// void main() {
+  ///     var bot = Nyxx("TOKEN");
+  ///     Map<String, dynamic> rawJson = /* rawJson from the API */
+  ///     DMChannel dmChannel = EntityUtility.createGuildMessage(bot, rawJson);
+  /// }
+  /// ```
+  static GuildMessage createGuildMessage(
+      INyxx client, Map<String, dynamic> rawJson) =>
+      GuildMessage._new(client, rawJson);
```

Comment:
its pretty much useless if there is createMessage

Reply:
done